src/script/services/publish/web-publish.ts

```diff
@@ -13,33 +14,41 @@ export async function generateWebPackage() {
     const chosenSW = getChosenServiceWorker();
 
     // The web package generator dies when screenshots is null. If detected, set screenshots to empty array.
-    const manifestWithScreenshots = manifest ? { ...manifest } : { ...genManifest };
+    const manifestWithScreenshots = manifest
+      ? { ...manifest }
+      : { ...genManifest };
 
     if (!manifestWithScreenshots.screenshots) {
       manifestWithScreenshots.screenshots = [];
     }
 
-    let urlToUse;
+    let baseUrl: string;
+    let urlToUse: string;
+    let headers: Headers | undefined;
+    let body: FormData | string;
 
-    if (chosenSW) {
-      urlToUse = `${env.webPackageGeneratorUrl}?siteUrl=${url
-      }&swId=${chosenSW}&hasServiceWorker=${false}`
+    if (manifest.icons.length > 10) {
```

Comment:
Should we just always default to using form post? I'm concerned about cases where there are few images, but they're large. Seems like multipart form post is the best way to go when dealing with potentially large files.